lib/additional_methods.rb

```diff
@@ -0,0 +1,24 @@
+
+class String
+  def without_quotes
```

Comment:
It's calling "monkey-patching". Use it carefully because you are extending a behavior of system classes. I would prefer to create some the utilities module.